src/transferlistwidget.cpp

```diff
@@ -424,6 +424,19 @@ void TransferListWidget::copySelectedMagnetURIs() const
     qApp->clipboard()->setText(magnet_uris.join("\n"));
 }
 
+// create list of captions from tasklist.
+void TransferListWidget::copySelectedCaptions() const
```

Comment:
Why "Copy captions" in the function name, coments ...? I think should be "Copy name" as in the displayed text.